testing/it/e2e-selenium/src/test/java/com/adobe/cq/wcm/core/components/it/seljup/tests/accordion/v1/AccordionIT.java

```diff
@@ -173,9 +175,12 @@ private ElementsCollection createItem() throws InterruptedException {
         assertTrue(items.get(0).getValue().equals("item0"), "First input item should be item0");
         assertTrue(items.get(1).getValue().equals("item1"), "Second input item should be item1");
         assertTrue(items.get(2).getValue().equals("item2"), "Third input item should be item2");
+
+        List<String> itemValues = items.stream().map(SelenideElement::getValue).map(String::toString).collect(Collectors.toList());
+
         Commons.saveConfigureDialog();
 
-        return items;
+        return itemValues;
```

Comment:
`ElementsCollection` becomes empty after the configuration dialog is closed, returning the values as copied into a string list.